POS_Image-RPi-Bootstrap/rpm-properties.xml

```diff
@@ -0,0 +1,11 @@
+<props>
+  <summary>Bootable Raspberry Pi SD Card image with Saltboot initrd</summary>
+  <description>
+    Bootable Raspberry Pi SD Card image with Saltboot initrd
+  </description>
+  <license>GPL-2.0</license>
+  <licensefile>LICENSE</licensefile>
```

Comment:
I don't know if this can cause a build failure, but you are missing LICENSE file in this PR.